packages/components/action-menu/src/index.less

```diff
@@ -1,4 +0,0 @@
-@import '~@ovh-ux/ui-kit.core/src/index';
-@import '~@ovh-ux/ui-kit.button/src/index';
-@import '~@ovh-ux/ui-kit.dropdown/src/index';
-@import './less/action-menu';
```

Comment:
I'm having trouble understanding the removal of the less file. Shouldn't the component still import its style to be standalone ?
`@import './less/action-menu';`